src/node_util.cc

```diff
@@ -379,27 +380,45 @@ void Initialize(Local<Object> target,
         .Check();
   }
 
-#define V(name)                                                               \
-  target->Set(context,                                                        \
-              FIXED_ONE_BYTE_STRING(env->isolate(), #name),                   \
-              Integer::New(env->isolate(), Promise::PromiseState::name))      \
-    .FromJust()
-  V(kPending);
-  V(kFulfilled);
-  V(kRejected);
+  {
+    Local<ObjectTemplate> tmpl = ObjectTemplate::New(isolate);
```

Comment:
Why are we using an `ObjectTemplate` here? I think it would be alright even if we just use a plain object here?

Reply:
Updated it with a plain object. Used it to reduce Check() calls and the parameters.

I thought it was a convention to use `ObjectTemplate` only when instantiating a complex Object or setting property/index handlers in detail. So I also had a little doubt whether we could use it here also once seeing `private symbols` are set using it.

Reply:
Oh, I see, prívate symbols have to be installed on an ObjectTemplate because the v8::Private are v8::Data but not v8::Value. There is no such complexity with other primitives and using plain objects for them is the convention.

Reply:
Aha.. right. Thanks for letting me know.